src/templates/below-component.js

```diff
@@ -0,0 +1,26 @@
+const { stylingTypes } = require('../helpers');
+
+const styled = `\
+const Wrapper = styled.div\`
+\`;
```

Comment:
Nit: can you add a blank line inside the template string?

Will Prettier strip it out, if you do? I like the idea of having a space for someone to add their styles, but it's not a big deal :)

Reply:
Yeah, Prettier will make it like so:
`const Wrapper = styled.div```

Reply:
ok! No problem then :)